config/samples/go_example_loadtest.yaml

```diff
@@ -50,6 +50,12 @@ spec:
   # results:
   #   bigQueryTable: "example-project.foo.demo_dataset"
 
+  # timeout and ttl field only take numbers and converted to duration, the unit
+  # of duration is fixed as ns, for example: 1s = 1000000000 ns, the number
+  # below indicate timeout=30s, ttl=2min
```

Comment:
I like that we add these to the examples, but I wish we could elaborate a little more on each one.

Maybe we could break them into each having their own comments and explain what a TTL or timeout does to effect a test.

As a developer, I feel like I may be confused to know what to put for both of them. It may be helpful to specify the timeout is the maximum amount of time a test is allowed to run. Meanwhile, the TTL is the amount of time the load test and its result status should live in memory before being garbage collected.

If I was using this framework for the first time, I probably would not be likely to check the docs for these fields. I'd probably just play around with these examples and read their comments.

Reply:
We should also add them to other languages.

Reply:
Yes, this just is an example for us to discuss if we want to do it the way it is now, once we are sure about this, I will add them to all other languages (go is the easiest/fastest to try). Thanks for reminding me!

For the explanation, I will separate them and explain more for each.